src/collective/cover/tiles/calendar.py

```diff
@@ -169,12 +177,10 @@ def isToday(self, day):
             self.now[2] == day and self.now[1] == self.month and self.now[0] == self.year)
 
     def getReviewStateString(self):
-        states = self.calendar.getCalendarStates()
-        return ''.join(map(lambda x: 'review_state={0}&amp;'.format(self.url_quote_plus(x)), states))
+        return 'review_state=published&amp;'
```

Comment:
@rodfersou do not hard-code this; all content types providing `plone.event.interfaces.IEvent` must be treated as events.

see: https://community.plone.org/t/canonical-way-of-declaring-a-content-type-as-event-like/1735?u=hvelarde